### PR TITLE
Fixed syntax breakdown of TxOut in building a transaction

### PR DESCRIPTION
**Pull request by osephson**
- Fix the syntax breakdown of `--tx-out` in building a raw transaction.
- In addition, changed the `--network` of `transactionCalculateMinFee` method due to its network.

### CODE REVIEW

File: helper.js [hunk 1]
```diff
@@ -135,14 +135,18 @@ exports.txInToString = (dir, txInList, isCollateral) => {
 
 exports.txOutToString = (txOutList) => {
   let result = "";
+  let assetOutStr;
   txOutList.forEach((txOut) => {
-    result += `--tx-out "${txOut.address}+${txOut.value.lovelace}`;
+    assetOutStr = ""
```
Comment (miguelaeh) [inline helper.js]:
```suggestion
    assetOutStr = "";
```

File: helper.js [hunk 2]
```diff
@@ -135,14 +135,18 @@ exports.txInToString = (dir, txInList, isCollateral) => {
...
+    result += `--tx-out ${txOut.address}+${txOut.value.lovelace}`;
     Object.keys(txOut.value).forEach((asset) => {
       if (asset == "lovelace") return;
-      result += `+${txOut.value[asset]} ${asset}`;
+      assetOutStr += `+${txOut.value[asset]} ${asset}`;
     });
-    result += `" `;
-    txOut.datumHash && (result += `--tx-out-datum-hash ${txOut.datumHash}`);
+    if (assetOutStr)
```
Comment (miguelaeh) [inline helper.js:146]: Is this check actually necessary? If `assetOutStr` is empty, the concat will remain the same right?

File: helper.js [hunk 3]
```diff
@@ -135,14 +135,18 @@ exports.txInToString = (dir, txInList, isCollateral) => {
...
+      result += `+"${assetOutStr.slice(1)}"`;
+    result += ' ';
+    txOut.datumHash && (result += `--tx-out-datum-hash ${txOut.datumHash} `);
```
Comment (miguelaeh) [inline helper.js]:
```suggestion
    txOut.datumHash && (result += ` --tx-out-datum-hash ${txOut.datumHash}`);
```

Comment (miguelaeh) [reply helper.js]: I guess the final space is not actually needed, but not fully sure

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thanks for taking the time to submit the PR!!

File: helper.js (hunk 2 above)
Comment (osephson) [reply helper.js:146]: If `assetOutStr` is empty, `result` will end with `+""`. If it is acceptable, it's okay to remove the check.

Comment (miguelaeh) [reply helper.js:146]: mmm that's true, I am not sure if that is a valid syntax. so let's leave it for safe

**Review by miguelaeh — APPROVED** [review]
LGTM!

**Review by miguelaeh — COMMENTED** [review]
BTW could you bump the version please?

**Comment by osephson** [comment]
Bumped version as `4.1.2`. PTAL, @miguelaeh !

**Review by miguelaeh — APPROVED** [review]

**Pull request merged**